validator/project_validator.go

```diff
@@ -662,7 +662,21 @@ func validateBVFields(project *model.Project) ValidationErrors {
 					break
 				}
 			}
-			if !taskHasValidDistro {
+			if !taskHasValidDistro && task.IsGroup {
+				taskGroupTasksHasDistro := true
+				for _, t := range project.FindTaskGroup(task.Name).Tasks {
+					pt := project.FindProjectTask(t)
+					if pt != nil {
+						for _, d := range pt.RunOn {
+							if d == "" {
```

Comment:
is there a reason why the `if d != "" {set to true}` logic won't work here? The false case is actually most likely to happen when run_on is completely empty which isn't captured here (and should be captured by the test if it's not already)

Reply:
it was because we need every task in the group to have a valid runOn if it's not specified in the bv or group instead of just checking 1 but i can work around it